src/data/faq_de.json

```diff
@@ -87,6 +87,125 @@
+                        ]
+                    },
+                    {
+                        "title": "Werden Warnung aufgrund eines Schnelltests oder PCR-Tests unterschieden?",
+                        "anchor": "rat_warning_not_differentiated",
+                        "active": true,
+                        "textblock": [
+                            "Kontaktpersonen erhalten in ihrer Corona-Warn-App eine Warnung über eine Begegnung an einem Tag mit niedrigem (grüne Kachel) oder erhöhtem Risiko (rote Kachel). Ob die Warnung aufgrund eines Schnelltests oder aufgrund eines PCR-Tests erfolgte, ist in der App nicht ersichtlich."
+                        ]
+                    },
+                    {
+                        "title": "EFGS Anbindung? Werden positive Ergebnisse auch in anderen Ländern geteilt?",
+                        "anchor": "rat_efgs",
+                        "active": true,
+                        "textblock": [
+                            "Die Warnungen, die auf einem Schnelltest-Ergebnis basieren, werden nicht über das EFGS (European Federation Gateway Service) mit anderen EU-Ländern geteilt, auch nicht mit der Schweiz, da die Nutzungsbedingungen des EFGS nicht vorsehen, dass Schnelltest-Warnungen geteilt werden."
+                        ]
+                    },
+                    {
+                        "title": "Was ist der Unterschied zwischen PCR-Tests, Antigen-Schnelltests und Selbsttests?",
+                        "anchor": "rat_differences_pcr_rat",
+                        "active": true,
+                        "textblock": [
+                            "PCR-Tests sind der 'Goldstandard' unter den Corona-Tests. Die Probenentnahme erfolgt durch medizinisches Personal – die Auswertung durch Labore.",
+                            "Antigen-Schnelltests: Haben ihren Namen, weil das Ergebnis schnell vorliegt. Sie können nur von geschultem Personal durchgeführt werden. Dafür wird ähnlich wie beim PCR-Test ein Nasen- oder Rachenabstrich gemacht. Die Auswertung erfolgt im Gegensatz zu den PCR-Test aber direkt vor Ort. Seit dem 8. März 2021 hat jeder Anspruch auf mindestens einen Schnelltest pro Woche.",
+                            "Selbsttests: Haben ihren Namen, weil jeder diese Tests selber machen kann, zum Beispiel zu Hause. Die Selbsttests sind zur Anwendung durch Privatpersonen bestimmt. Dafür muss die Probenentnahme und -auswertung entsprechend einfach sein. Der Test kann zum Beispiel mit einem Nasenabstrich oder mit Speichel erfolgen. Das Bundesinstitut für Arzneimittel und Medizinprodukte überprüft zusammen mit dem Paul-Ehrlich-Institut Qualität und Aussagekraft der Tests.",

```

Comment:
Add here that Selbsstest can't be shared?

Reply:
This is just an explanation of different tests. Yes, we need another FAQ entry stating that Selbsttests can't be used within the CWA